ucm2/codecs/wcd934x/HeadphoneDisableSeq.conf

```diff
@@ -1,6 +1,8 @@
-cset "name='COMP1 Switch' 0"
-cset "name='COMP2 Switch' 0"
-cset "name='RX1 Digital Volume' 0"
-cset "name='RX2 Digital Volume' 0"
-cset "name='RX INT1 DEM MUX' ZERO"
-cset "name='RX INT2 DEM MUX' ZERO"
+DisableSequence [
+	cset "name='COMP1 Switch' 0"
+	cset "name='COMP2 Switch' 0"
+	cset "name='RX1 Digital Volume' 0"
```

Comment:
Volume controls should be moved to the boot sequences.

Reply:
Okay will remove the volume setup from this disable path totally.